Index: app/src/main/java/com/example/hunterqrhunter/page/QRMapScreen.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.hunterqrhunter.page;\n\nimport android.os.Bundle;\n\nimport androidx.annotation.Nullable;\nimport androidx.appcompat.app.AppCompatActivity;\n\nimport com.example.hunterqrhunter.R;\n\npublic class QRMapScreen extends AppCompatActivity {\n    @Override\n    protected void onCreate(@Nullable Bundle savedInstanceState) {\n        super.onCreate(savedInstanceState);\n        setContentView(R.layout.activity_qr_map);\n    }\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/hunterqrhunter/page/QRMapScreen.java b/app/src/main/java/com/example/hunterqrhunter/page/QRMapScreen.java
--- a/app/src/main/java/com/example/hunterqrhunter/page/QRMapScreen.java	(revision 1c3db61066041abca4dd8300cff3388745140e87)
+++ b/app/src/main/java/com/example/hunterqrhunter/page/QRMapScreen.java	(date 1678678433900)
@@ -13,4 +13,4 @@
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_qr_map);
     }
-}
+}
\ No newline at end of file
Index: settings.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>pluginManagement {\n    repositories {\n        google()\n        mavenCentral()\n        gradlePluginPortal()\n    }\n}\ndependencyResolutionManagement {\n    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)\n    repositories {\n        google()\n        mavenCentral()\n    }\n}\ninclude ':app'\nrootProject.name = \"HunterQRHunter\"\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/settings.gradle b/settings.gradle
--- a/settings.gradle	(revision 1c3db61066041abca4dd8300cff3388745140e87)
+++ b/settings.gradle	(date 1678678149066)
@@ -12,5 +12,5 @@
         mavenCentral()
     }
 }
-include ':app'
 rootProject.name = "HunterQRHunter"
+include ':app'
