Index: settings.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>pluginManagement {\n    repositories {\n        google()\n        mavenCentral()\n        gradlePluginPortal()\n    }\n}\ndependencyResolutionManagement {\n    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)\n    repositories {\n        google()\n        mavenCentral()\n    }\n}\ninclude ':app'\nrootProject.name = \"HunterQRHunter\"\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/settings.gradle b/settings.gradle
--- a/settings.gradle	(revision 023e073f4e4f7df7c4ee720df06fb8daa65890e5)
+++ b/settings.gradle	(date 1678679946647)
@@ -12,5 +12,6 @@
         mavenCentral()
     }
 }
-include ':app'
 rootProject.name = "HunterQRHunter"
+include ':app'
+
